### PR TITLE
veml6070: add Vishay UV light sensor

### PR DESCRIPTION
**Pull request by cmur2**
Product: https://www.vishay.com/docs/84277/veml6070.pdf

I would like to contribute support for this UV light sensor to use it easily with tinygo, I tested the submitted code on the sensor I own and it works well. I used this sensor before which is why I created https://github.com/cmur2/python-veml6070

Please give me feedback for this PR, should I change something or is there more documentation to add?

### CODE REVIEW

File: veml6070/veml6070.go [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+package veml6070 // import "tinygo.org/x/drivers/veml6070"
+
+import (
+	"time"
+
+	"machine"
+)
+
+// Device wraps an I2C connection to a VEML6070 device.
+type Device struct {
```
Comment (conejoninja) [inline veml6070/veml6070.go]: We usually allow the I²C address to be configurable (for example https://github.com/tinygo-org/drivers/blob/master/bh1750/bh1750.go#L20 ), but this case is a bit special, as it uses two address for reading. Maybe it's a good idea to allow both to be configurable, or maybe configure one and let the other be +1. What do @aykevl & @deadprogram  think?

File: veml6070/veml6070.go [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+	return d.bus.Tx(uint16(ADDR_L), []byte{CONFIG_DISABLE}, nil)
+}
+
+func (d *Device) enable() error {
+	return d.bus.Tx(uint16(ADDR_L), []byte{CONFIG_ENABLE | d.IT}, nil)
+}
+
+func (d *Device) readData(address uint16) (byte, error) {
+	data := []byte{0}
+
```
Comment (conejoninja) [inline veml6070/veml6070.go]: for my taste, there's no need for these empty lines

File: veml6070/veml6070.go [hunk 3]
```diff
@@ -0,0 +1,138 @@
+// Package bmp180 provides a driver for the BMP180 digital pressure sensor
```
Comment (conejoninja) [inline veml6070/veml6070.go]: References to BMP180 digital pressure sensor

File: veml6070/veml6070.go (hunk 1 above)
Comment (cmur2) [reply veml6070/veml6070.go]: I also thought about this but then this sensor only allows those two fixed addresses to be used - should it still be configurable?

File: veml6070/veml6070.go (hunk 3 above)
Comment (cmur2) [reply veml6070/veml6070.go]: Uh thanks for spotting my copy&paste error :+1:

File: veml6070/veml6070.go (hunk 2 above)
Comment (cmur2) [reply veml6070/veml6070.go]: Removed

File: veml6070/veml6070.go (hunk 1 above)
Comment (deadprogram) [reply veml6070/veml6070.go]: I think we need to expose the same interface, but what the driver actually does can be specific.

One reason we have the `Address` able to be changed, is to handle "clone" or "off-market" sensors that use different addresses. Another is the case that someone uses an I2C multiplexer like https://www.adafruit.com/product/2717

Basically, it seems best to allow it to be changed, even if the device itself does not.

Comment (cmur2) [reply veml6070/veml6070.go]: That's a good point, didn't thinkg about that... I now made both (the lower and the higher) addresses individually configurable so the user has maximum freedom - what do you think @deadprogram ?

Comment (deadprogram) [reply veml6070/veml6070.go]: The change looks good to me.

File: README.md [hunk 4]
```diff
@@ -82,6 +82,7 @@ func main() {
 | [ST7735 TFT color display](https://www.crystalfontz.com/controllers/Sitronix/ST7735R/319/) | SPI |
 | [ST7789 TFT color display](https://cdn-shop.adafruit.com/product-files/3787/3787_tft_QT154H2201__________20190228182902.pdf) | SPI |
 | [Thermistor](https://www.farnell.com/datasheets/33552.pdf) | ADC |
+| [VEML6070](https://www.vishay.com/docs/84277/veml6070.pdf) | I2C |
```
Comment (deadprogram) [inline README.md]: This should probably say "VEML6070 UV light sensor" not just "VEML6070".

Comment (cmur2) [reply README.md]: You're right, done! Anything else?

**Comment by cmur2** [comment]
Is this PR mergeble now?

**Comment by deadprogram** [comment]
I've switched the branch for the PR to `dev`, and will now merge it. Thank you @cmur2 for the contribution!

**Pull request merged**